src/Xamarin.Android.Build.Tasks/Tasks/CollectNonEmptyDirectories.cs

```diff
@@ -22,7 +22,7 @@ public override bool Execute ()
 				var firstFile = Directory.EnumerateFiles(directory.ItemSpec, "*.*", SearchOption.AllDirectories).FirstOrDefault ();
 				if (firstFile != null) {
 					var taskItem = new TaskItem (directory.ItemSpec, new Dictionary<string, string> () {
-						{"FileFound", firstFile}
+						{"FileFound", Path.GetFullPath (Path.Combine (directory.ItemSpec, "..", "..")) + ".stamp" }
```

Comment:
Maybe this should just add a new metadata for `%(StampFile)`? And keep `%(FileFound)`?

Reply:
Would need to update `Xamarin.Android.Common.targets` to use `%(StampFile)` if it was renamed.

Reply:
done